docs/api/platform/MbedStats.md

```diff
@@ -1,27 +1,75 @@
 ## Mbed statistics
 
-Mbed OS provides a set of functions that you can use to capture the memory and thread statistics at runtime. `mbed_stats.h` declares these functions. To enable all Mbed OS statistics, you must build code with the `MBED_ALL_STATS_ENABLED` macro.
+Mbed OS provides a set of functions that you can use to capture the memory and thread statistics at runtime. `mbed_stats.h` declares these functions. To enable all Mbed OS platform statistics, you must enable the following Mbed OS configuration option:
+
```

Comment:
Query: Is this information already in the configuration section? If so, instead of duplicating it, can we link to it? If not, why not?

Reply:
It is sort of, it'd require some reading and understanding of how to setup the overrides in JSON. I could just say 
> enable `platform.cpu-stats-enabled`

but think this is a bit easier for  the user. It does add more things to the page though.

Reply:
We should definitely link to the config page, at least.

Reply:
Please share which config page? Also please new config will not be visible anywhere till code is merged

Reply:
This one: https://os-doc-builder.test.mbed.com/docs/development/config/configuration-platform.html